frontend/src/container/OnboardingContainer/Steps/EnvironmentDetails/EnvironmentDetails.tsx

```diff
@@ -122,6 +126,18 @@ export default function EnvironmentDetails(): JSX.Element {
 					) {
 						return null;
 					}
+					if (
+						selectedModule?.id !== useCases.APM.id &&
+						environment.id === 'docker'
+					) {
+						return null;
+					}
+					if (
```

Comment:
This looks same as the above if condition.

Reply:
Have added a Docker tile in the environment selection for APM only that's why this condition.
The above one is for **kubernetes**, this one is for **docker**.

Reply:
There are 2 if blocks for docker

Reply:
Sorry. My bad. Fixed this.